bucket/torrenttools.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.6.2",
+    "description": "Commandline tool for inspecting, creating and editing BitTorrent metafiles.",
+    "homepage": "https://github.com/fbdtemme/torrenttools",
+    "license": {
+        "identifier": "MIT"
+    },
+    "url": "https://github.com/fbdtemme/torrenttools/releases/download/v0.6.2/torrenttools-0.6.2-windows-x86_64.msi",
```

Comment:
For 64 bit applications, we prefer them being present in an architecture/64bit block

Check Mage for example

https://github.com/ScoopInstaller/Main/blob/master/bucket/mage.json#L6

Same for autoupdate block